src/stirling/source_connectors/socket_tracer/protocols/pgsql/stitcher.cc

```diff
@@ -432,9 +432,6 @@ RecordsWithErrorCount<pgsql::Record> StitchFrames(std::deque<pgsql::RegularMessa
   while (req_iter != reqs->end() && resp_iter != resps->end()) {
     auto& req = *req_iter;
     ++req_iter;
-    if (resp_iter->consumed) {
-      continue;
-    }
```

Comment:
I added a `CHECK(false)` statement here and re ran the bpf test 20 times to verify that this logic was never triggered.

This was identified as part of @etep's code review.